assets/home/notices.js

```diff
@@ -130,11 +130,9 @@ const Notice = ( { noticeId, notice, dismissNonce } ) => {
 
 	return (
 		<div { ...containerProps }>
-			<SenseiCircleLogo className="sensei-notice__icon" />
-			<div className="sensei-notice__wrapper">
-				<div className="sensei-notice__content">
-					<RawHTML>{ message }</RawHTML>
-				</div>
+			<div className="sensei-notice__content">
```

Comment:
I changed it to have a wrapper getting the icon and text. So we can break the other parts depending on the screen size.